server/src/internalClusterTest/java/org/opensearch/search/aggregations/metrics/CardinalityIT.java

```diff
@@ -615,5 +639,6 @@ public void testScriptCaching() throws Exception {
                 .getMissCount(),
             equalTo(2L)
         );
+        assertAcked(client().admin().indices().delete(new DeleteIndexRequest("cache_test_idx")).get());
```

Comment:
nit: instead of deleting like this we can use `cluster().wipeIndices("cache_test_idx")`. Same for other places.